odoo/http.py

```diff
@@ -147,13 +149,18 @@
 except ImportError:
     from werkzeug.contrib.fixers import ProxyFix
 
+try:
+    from werkzeug.utils import send_file as werkzeug_send_file
+except ImportError:
+    from .tools._vendor.send_file import send_file as werkzeug_send_file
```

Comment:
Seems to have gone missing in action?

And why not keep it as `send_file` since the old one has been removed?

Reply:
I want `from odoo.http import send_file` to fail. I fixed the code to correctly use `werkzeug_send_file` (renamed `_send_file`)